keylime/tpm/tpm_abstract.py

```diff
@@ -230,21 +230,37 @@ def __check_ima(self, agent_id, pcrval, ima_measurement_list, allowlist, ima_key
         logger.debug(f"IMA measurement list of agent {agent_id} validated")
         return True
 
-    def check_pcrs(self, agent_id, tpm_policy, pcrs, data, virtual, ima_measurement_list, allowlist, ima_keyring, mb_measurement_list, mb_intended_state):
+    def check_pcrs(self, agent_id, tpm_policy, pcrs, data, virtual, ima_measurement_list, allowlist, ima_keyring, mb_measurement_list, mb_refstate):
         try:
             tpm_policy_ = ast.literal_eval(tpm_policy)
         except ValueError:
             tpm_policy_ = {}
         pcr_allowlist = tpm_policy_.copy()
 
-        if mb_measurement_list or mb_intended_state :
+        if mb_measurement_list or mb_refstate :
             logger.info("Measured boot information received, but for now it will not be processed. A future update will enable the full processing of it.")
 
         if 'mask' in pcr_allowlist:
             del pcr_allowlist['mask']
         # convert all pcr num keys to integers
         pcr_allowlist = {int(k): v for k, v in list(pcr_allowlist.items())}
 
+        if mb_refstate not in (False, None) and mb_measurement_list not in (False, None) :
```

Comment:
You shouldn't need the False or None check, just checking it's "truthiness" should work:

`if mb_refstate and mb_measurement_list`

Reply:
there was reason for this (we discussed the possibility of admitting an empty dictionary as a reference state), but decided against if for this PR